src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -754,6 +758,49 @@ public override bool Equals(ConstantValue? other)
             }
         }
 
+        private sealed class ConstantValueNativeInt : ConstantValueDiscriminated
```

Comment:
>ConstantValueNativeInt [](start = 29, length = 22)

Should we just re-use `ConstantValueI32` or share the code in some way?

Reply:
I considered that but decided against it since we're not sharing code across other concrete `ConstantValue` types.

---
In reply to: [379035536](https://github.com/dotnet/roslyn/pull/41380#discussion_r379035536) [](ancestors = 379035536)